src/sentry/search/events/filter.py

```diff
@@ -1129,3 +1131,17 @@ def _issue_filter_converter(self, search_filter: SearchFilter) -> Optional[Where
                 SearchValue(filter_values if search_filter.is_in_filter else filter_values[0]),
             )
         )
+
+    def _transaction_status_filter_converter(
+        self, search_filter: SearchFilter
+    ) -> Optional[WhereType]:
+        converted = _transaction_status_filter_converter(
```

Comment:
The plan is to burn these old functions entirely so we no longer construct arrays for conditions anymore, can we re-implement this function here instead of depending on the existing code?

Reply:
Sounds good, done!